clients/da-vinci-client/src/main/java/com/linkedin/davinci/schema/merge/SortBasedCollectionFieldOpHandler.java

```diff
@@ -773,7 +773,7 @@ private UpdateResultStatus handleModifyPutOnlyMap(
     deletedKeys.sort(String::compareTo);
 
     // Step 3: Set new active elements and their active timestamps.
-    setNewMapActiveElementAndTs(
+    setNewMapActiveElementAndTimestamp(
```

Comment:
Nitpick... I would prefer trivial renames to be in a separate PR, to minimize the size of complex PRs... but up to you.